PostHog/PostHogContext.swift

```diff
@@ -225,23 +230,25 @@ class PostHogContext {
         if let deviceType = staticCtx["$device_type"] {
             personProperties["$device_type"] = deviceType
         }
-        if let deviceManufacturer = staticCtx["$device_manufacturer"] {
-            personProperties["$device_manufacturer"] = deviceManufacturer
-        }
-        if let deviceModel = staticCtx["$device_model"] {
-            personProperties["$device_model"] = deviceModel
-        }
-
-        // Localization - read directly to avoid expensive dynamicContext call
-        if #available(iOS 16.0, macOS 13.0, tvOS 16.0, watchOS 9.0, *) {
-            if let languageCode = Locale.current.language.languageCode {
-                personProperties["$locale"] = languageCode.identifier
-            }
-        } else {
-            if let languageCode = Locale.current.languageCode {
-                personProperties["$locale"] = languageCode
-            }
-        }
```

Comment:
https://github.com/PostHog/posthog-android/pull/315/files#r2495899139

Reply:
they were not added since default flags props didnt exist, you can check the PR above